net/http/partitioned_host_state_map.h

```diff
@@ -37,8 +37,7 @@ class NET_EXPORT PartitionedHostStateMapBase {
   bool HasPartitionHash() const;
   // Returns true if |partition_hash_| contains a non empty valid hash.
   bool IsPartitionHashValid() const;
-  // Creates a host hash by concatenating first 16 bytes (half of SHA256) from
-  // |k| and first 16 bytes from |partition_hash_|.
+  // Returns first `HashedHost::size() / 2` bytes from |k|.
```

Comment:
this comment should go near `GetHalfKey`

Reply:
Oops, fixed in eb96a2aa698459aacbf30e3e2196b95e0093d33b.